bmeg_app/views/literature_support_view.py

```diff
@@ -48,90 +44,134 @@
-        tooltip_data=[{column: {'value': str(value), 'type': 'markdown'} for column, value in row.items()} for row in df.to_dict('records')],
+        #tooltip_data=[{column: {'value': str(value), 'type': 'markdown'} for column, value in row.items()} for row in df.to_dict('records')],
         tooltip_duration=None,
         page_size=10,
     )
-    return dashtable,
-
-@app.callback(
-    Output("resp_histo", "children"),
-    [Input('gene_dd', 'value')]
-)
-def render_callback(selected_gene):
-    '''pie chart taxonomy'''
-    # Filter for gene
-    df = base_df[base_df['geneID']==selected_gene].reset_index(drop=True)
-    # Generate figure
-    fig= lsu.get_histogram_side(base_df['response'],main_colors['pale_yellow'])
-    return html.Div(dcc.Graph(figure=fig)),
 
-@app.callback(
-    Output("pie_taxon", "children"),
-    [Input('gene_dd', 'value')]
-)
-def render_callback(selected_gene):
-    '''pie chart taxonomy'''
-    # Filter for gene
-    df = base_df[base_df['geneID']==selected_gene].reset_index(drop=True)
-    # Generate figure
-    fig= lsu.pie_from_dict(lsu.count_taxonomy(df),False)
-    return html.Div(dcc.Graph(figure=fig)),
+    citations = G.query().V(gene).out("g2p_associations").as_("a").out("publications").as_("p").select("a").out("compounds").as_("c").aggregate(gripql.term("chem_citation", "$c.synonym")).execute()
+    citation_counts = []
+    for i in citations[0]["chem_citation"]["buckets"]:
+        citation_counts.append( { "compound" : i['key'], "citations" : i['value'] } )
 
-@app.callback(
-    Output("occr", "children"),
-    [Input('gene_dd', 'value')]
-)
-def render_callback(selected_gene):
-    '''Occurance table'''
-    # Filter for gene
-    df = base_df[base_df['geneID']==selected_gene].reset_index(drop=True)
-    # Generate table
-    df = lsu.occurances_table(df,'drug',['Compound','Lit Occurances'])
-    dashtable = dash_table.DataTable(
+    pub_table = dash_table.DataTable(
         id='export_pub_table',
-        data = df.to_dict('records'),
-        columns=[{"name": i, "id": i} for i in df.columns],
-        style_header={'text-align':'center','backgroundColor': 'rgb(230, 230, 230)','fontSize':styles['t']['size_font'],'fontWeight': 'bold','fontFamily':styles['t']['type_font']},
+        data = citation_counts[:10],
+        columns=[{"name": "Compound", "id": "compound"}, {"name" : "Citations", "id" : "citations"}],
+        style_header={'text-align':'left','backgroundColor': 'rgb(230, 230, 230)','fontSize':styles['t']['size_font'],'fontWeight': 'bold','fontFamily':styles['t']['type_font']},
         style_cell={'maxWidth':'100px','padding-left': '20px','padding-right': '20px'},
-        style_data={'whiteSpace':'normal','height':'auto','text-align':'center','fontFamily':styles['t']['type_font'],'fontSize':styles['t']['size_font']},
+        style_data={'whiteSpace':'normal','height':'auto','text-align':'left','fontFamily':styles['t']['type_font'],'fontSize':styles['t']['size_font']-2},
```

Comment:
why `-2` on font size? uniform and consistent styling might be best to keep as is `'fontSize':styles['t']['size_font']`

Reply:
Making side panel smaller to leave more space for main table